src/test/java/persistence/sql/entity/EntityManagerImplTest.java

```diff
@@ -89,6 +90,20 @@ void entityManager_persist() {
     }
 
 
+    @Test
+    @DisplayName("EntityManager의 persist시 idValue가 null인 경우")
```

Comment:
테스트의 설명을 작성해주실 때는 결과까지 함께 적어주시면 가독성에 도움이 될 것 같아요! 👍
```suggestion
    @DisplayName("EntityManager의 persist시 idValue가 null인 경우 id값이 자동으로 생성된다")
```

Reply:
네 앞으로 작성될 테스트에서는 케이스와 예상결과까지 작성하도록 해보겠습니다 👍